pkgs/applications/graphics/sane/backends/brscan4/preload.c

```diff
@@ -0,0 +1,170 @@
+/* Brgen4 search for configuration under `/etc/opt/brother/scanner/brscan4`. This
```

Comment:
Okay, seems complicated enough to probably not worth to tweak the existing libredir. Thanks for taking a look!